### ISSUE TITLE
RFE: attach /tmp/anaconda* files for Anaconda crashes

### ISSUE THREAD

**Issue by mschwendt** [abrt/abrt#999]
At https://bugzilla.redhat.com/1233177, an Anaconda developers complained about this:

> In the future, please attach the logs from /tmp to anaconda bugs, in particular
> the /tmp/anaconda-tb* file containing the full traceback data. The files you
> attached contain no useful information.

It was ABRT that attached those files. ABRT could be taught to attach the requested files.

**Comment by jfilak** [comment]
True, however, Anaconda does not use ABRT, but python-meh + libreport -> rhinstaller/python-meh#3

**Comment by jfilak** [comment]
D`oh, the bugzilla bug https://bugzilla.redhat.com/1233177 has been detected by ABRT (I did not notice you ran Anaconda from the live Fedora Workstation image).
